example/web-access-control/protected-operation/debug-direct-acl.feature

```diff
@@ -0,0 +1,33 @@
+Feature: Test ACL creation in apply mode
```

Comment:
OK, I have to admit that I haven't look at the actual ACR spec, as the draft hasn't yet been merged.

Reply:
Both spec and implementations are evolving so there will be more changes to come